src/Grpc.Auth/.gitignore

```diff
@@ -0,0 +1,3 @@
+bin
+obj
+*.nupkg
```

Comment:
Is this necessary? There is a .gitignore in the root folder

(I think this came up for the Grpc.Core.Api review)

Reply:
Removed.